### ISSUE TITLE
Wordpress Error on Upgrade to v1.13.0

### ISSUE THREAD

**Issue by broksonic21** [wildbit/postmark-wordpress#65]
Happens on multiple sites

Notice: Undefined variable: postmark in /app/wp-content/plugins/postmark-approved-wordpress-plugin/postmark.php on line 457
Fatal error: Uncaught Error: Call to a member function load_settings() on null in /app/wp-content/plugins/postmark-approved-wordpress-plugin/postmark.php:457

Admin then gets an email saying a terminal error existed.

**Comment by tflight** [comment]
I'm seeing this as well across multiple sites. Possibly related https://github.com/wildbit/postmark-wordpress/issues/39?

```
Error Details
=============
An error of type E_ERROR was caused in line 457 of the file /home/s21/html/wp-content/plugins/postmark-approved-wordpress-plugin/postmark.php. Error message: Uncaught Error: Call to a member function load_settings() on null in /home/s21/html/wp-content/plugins/postmark-approved-wordpress-plugin/postmark.php:457
Stack trace:
#0 /home/s21/html/wp-includes/class-wp-hook.php(287): pm_upgrade_completed(Object(Plugin_Upgrader), Array)
#1 /home/s21/html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#2 /home/s21/html/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#3 /home/s21/html/wp-admin/includes/class-plugin-upgrader.php(371): do_action('upgrader_proces...', Object(Plugin_Upgrader), Array)
#4 /home/s21/html/wp-admin/update.php(51): Plugin_Upgrader->bulk_upgrade(Array)
#5 {main}
 thrown

**Comment by EatonZ** [comment]
Same here:
```
PHP Fatal error:  Uncaught Error: Call to a member function load_settings() on null in wp-content\plugins\postmark-approved-wordpress-plugin\postmark.php:457
Stack trace:
#0 wp-includes\class-wp-hook.php(287): pm_upgrade_completed()
#1 wp-includes\class-wp-hook.php(311): WP_Hook->apply_filters()
#2 wp-includes\plugin.php(478): WP_Hook->do_action()
#3 wp-admin\includes\class-plugin-upgrader.php(371): do_action()
#4 wp-admin\includes\ajax-actions.php(4451): Plugin_Upgrader->bulk_upgrade()
#5 wp-includes\class-wp-hook.php(287): wp_ajax_update_plugin()
#6 wp-includes\class-wp-hook.php(311): WP_Hook->apply_filters()
#7 wp-includes\plugin.php(478): WP_Hook->do_action()
#8 wp-admin\admin-ajax.php(176): do_action()
#9 {main}
  thrown in wp-content\plugins\postmark-approved-wordpress-plugin\postmark.php on line 457
```

Site appears to be working fine, but please let me know if there is something that needs to be done to complete the upgrade manually.

**Comment by pgraham3** [comment]
Hi @EatonZ,

Thanks for the report! No further action is needed to complete the upgrade. WordPress recently introduced new error catching for plugins that revealed this upgrade error. The error is fixed in v1.13.2 of the Postmark for WordPress plugin, so future upgrades will not have this error moving forward.
